src/lib/openapi/spec/health-overview-schema.ts

```diff
@@ -14,16 +14,23 @@ export const healthOverviewSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['version', 'name'],
```

Comment:
Like with the other schema: are these the only required properties? Can everything else be undefined?